dart-impl/mpi/CMakeLists.txt

```diff
@@ -6,8 +6,6 @@ set(DASH_DART_IMPL_MPI_LIBRARY dart-mpi)
 
 set(DASH_DART_BASE_LIBRARY dart-base)
 
-include(${CMAKE_SOURCE_DIR}/CMakeExt/MPI.cmake)
-
```

Comment:
Because it is already called in CMakeList in the top level directory: https://github.com/dash-project/dash/blob/development/CMakeLists.txt#L127

I think DASH also requires information on MPI, right?